users/templates/about.html

```diff
@@ -0,0 +1,102 @@
+{% extends 'base.html'%}
+
+{% block content %}
+
+<section class="jumbotron text-center my-0 w-100">
+  <div class="container">
+    <h1 class="jumbotron-heading">Square Up Event Scheduler</h1>
+    <small class="text-muted">By the Purple Nut Team</small>
+  </div>
+</section>
+<div class="album py-5">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4">
+        <div class="card mb-4 shadow-sm">
+          <img class="cover" width="100%" height="auto" src="https://avatars2.githubusercontent.com/u/30354775?s=460&amp;v=4" alt="">
+          <div class="card-body">
+            <h3>Royce Schultz</h3>
+            <strong class="text-muted">Tech Lead</strong>
+            <p class="card-text">I like Python</p>
+            <a class="btn btn-sm btn-outline-secondary" href="https://github.com/royceschultz" target="_blank">
+              Github
+            </a>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="card mb-4 shadow-sm">
+          <img class="cover" width="100%" height="auto" src="https://avatars3.githubusercontent.com/u/29512702?s=460&v=4" alt="">
+          <div class="card-body">
+            <h3>Bradley Ma</h3>
+            <strong class="text-muted">Position</strong>
+            <p class="card-text">Bio</p>
+            <a class="btn btn-sm btn-outline-secondary" href="https://github.com/bradleyyma" target="_blank">
+              Github
+            </a>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="card mb-4 shadow-sm">
+          <img class="cover" width="100%" height="auto" src="https://avatars2.githubusercontent.com/u/49109757?s=460&v=4" alt="">
+          <div class="card-body">
+            <h3>Chace Trevino</h3>
+            <strong class="text-muted">Position</strong>
+            <p class="card-text">Bio</p>
+            <a class="btn btn-sm btn-outline-secondary" href="https://github.com/chacetrev10" target="_blank">
+              Github
+            </a>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="card mb-4 shadow-sm">
+          <img class="cover" width="100%" height="auto" src="https://avatars1.githubusercontent.com/u/29643144?s=460&v=4" alt="">
+          <div class="card-body">
+            <h3>Manuel Lindo</h3>
+            <strong class="text-muted">Position</strong>
+            <p class="card-text">Bio</p>
+            <a class="btn btn-sm btn-outline-secondary" href="https://github.com/dacostalindo" target="_blank">
+              Github
```

Comment:
Position: Aerospace/Software Engineer 
Bio: The Portuguese will conquer the world once again!